<launch>
    <node name="task_planner" pkg="mrobosub_planning" type="task_planner.py">
        <rosparam command="load" file="$(find mrobosub_planning)/param/competition_states.yaml" />
         <rosparam command="load" file="$(find mrobosub_planning)/param/globals.yaml" />
    </node>
</launch>